mendix/README.md

```diff
@@ -0,0 +1,39 @@
+# Mendix
+
+## Overview
+
+[Mendix][1] is a market [leading low-code IDE][2] for professional developers. A powerful visual model-driven development environment to build apps on the Mendix Platform.
+
+With Mendix, you can easily create, change, integrate, test and deploy your applications. Moreover, you can manage branch lines and security as well as extend your app with custom code using the built-in editors.
+
+In addition to the native monitoring capabilities out of the box, Mendix also supports integrations to leading 3rd party application performance management vendors such as DataDog. Refer to our documentation to see all our [supported integrations][3].
+
+The Mendix integration with DataDog enables you to consume and monitor your Mendix ecosystem. The [DataDog integration][4] from Mendix will enable you to monitor
+Mendix Runtime metrics, Java Virtual Machine (JVM) metrics, database, SaaS(Software as a Service) environment metrics and any custom metrics you choose to
+configure in Mendix Studio Pro.
+
+
+> **Note**:  This integration is applicable to Mendix Cloud & Mendix Cloud Dedicated deployment models.
+
+
+## Setup
+
+To enable the DataDog integration for Mendix applications running on Mendix cloud follow the instructions [here][6].
```

Comment:
```suggestion
To enable the Datadog integration for Mendix applications running on Mendix cloud follow the instructions [here][6].
```

Reply:
Please address this comment. Datadog should be spelt with only one uppercase D rather than two.